presto-docs/src/main/sphinx/release/release-0.287.rst

```diff
@@ -0,0 +1,102 @@
+=============
+Release 0.287
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in CTE reference node creation, where different CTEs may be incorrectly considered as the same CTE. :pr:`22515`
+* Fix bug with spilling in TopNRowNumber. :pr:`22281`
+* Fix plan canonicalization by canonicalizing plan node ids. :pr:`22033`
+* Fix problem when writing large varchar values to throw a user error when it exceeds internal limits. :pr:`22063`
+* Fix queries that filter with ``LIKE '%...%'`` over char columns. :pr:`22076`
+* Fix the regr_count, regr_avgx, regr_avgy, regr_syy, regr_sxx, and regr_sxy functions result to be null when the input data is null, not 0. :pr:`22112`
+* Fix an issue with heuristic CTE materialization strategy where incorrect CTEs were materialized. :pr:`22433`
+* Fix precision loss when timestamp yielded from ``from_unixtime(double)`` function. :pr:`21899`
+* Improve accuracy and performance of HyperLogLog functions. :pr:`21943`
+* Improve repeat function to create RunLengthEncodedBlock to improve performance. :pr:`21984`
+* Improve latency of materialized CTEs by scheduling multiple dependent subgraphs independently. :pr:`22205`
+* Add Heuristic CTE Materialization strategy which auto materialized expensive CTEs. This is configurable by setting ``cte_materialization_strategy`` to ``HEURISTIC`` or ``HEURISTIC_COMPLEX_QUERIES_ONLY``. (default ``NONE``). :pr:`21720`
+* Add :doc:`Prestissimo Developer Guide <../prestissimo>` topic to the Presto documentation. :pr:`21953`
+* Add session property ``default-view-security-mode`` to choose the default security mode for view creation. :pr:`21956`
+* Add a new plan canonicalization strategy ``ignore_scan_constants`` which canonicalizes predicates for both partitioned and non-partitioned columns in scan node. :pr:`21832`
+* Add a session property ``track_history_based_plan_statistics_from_complete_stages_in_failed_query`` to enable tracking hbo statistics from complete stages in failed queries. :pr:`20947`
```

Comment:
``track_history_based_plan_statistics_from_complete_stages_in_failed_query`` is not documented in [Properties Reference](https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/admin/properties.rst).